honestcash-v2/src/app/modules/editor/converters/json-to-html.ts

```diff
@@ -175,3 +182,7 @@ export const convertBlockToHtml = (block: Block): string | null => {
     }
   }
 };
+
+export const getBlocksArrayLength = (blocks: Block[]): number => {
```

Comment:
We should have only one export per component. This code does not belong here to this file -> refactor.